Thesis.tex

```diff
@@ -222,11 +221,68 @@ \subsection{Reasonable and non-discriminatory licensing}
  
 Another important case related to RAND is Microsoft versus Motorola. In 2012, the United States Court of Appeals for the Ninth Circuit ruled that a patent holder's agreement to a policy like RAND creates a legally binding contract \parencite{MicrosoftVsMotorola}.
 
+\chapter{Discourse}
+\label{ch:Disc}
+
+\section{Copyright in the context of the arts}
+\label{sec:CopyArts}
+
+\subsection{Pro IP}
+
+\subsection{Against IP}
+
+\subsection{Addressing Against IP's points}
+
+\subsection{Adressing Pro IP's points}
+
+\section{Copyright in the context of the creation of software}
+\label{sec:CopySoft}
+
+\subsection{Pro IP}
+An average software user is not a developer and in many markets does not have technical expertise related to the workings of software they use. For those users a software product is just a tool they integrate into their work-flow. As a commercial user their time and productivity has a direct cost. Therefore many users hold training, documentation and support in high regards. Proprietary software vendors have a high incentive to offer training and support. A seizable potion of the cost to acquire and license their products is for support. 
```

Comment:
instead of a "commercial user", perhaps it would be better to use "business end user". At least I think this is the kind of user you think of right?